From 07fa166f1ebc4f053e96b0ce274c4aba74047cbb Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 28 Dec 2017 16:30:56 +0100
Subject: [PATCH] device.mk: Drop OnePlusDoze

device.mk: Drop OnePlusDoze APK that we don't need anyway.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 device.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/device.mk b/device.mk
index 9a8936f..ef7732a 100644
--- a/device.mk
+++ b/device.mk
@@ -108,10 +108,6 @@ PRODUCT_PACKAGES += \
     librmnetctl \
     rmnetcli
 
-# Doze
-PRODUCT_PACKAGES += \
-    OneplusDoze
-
 # Graphics
 PRODUCT_PACKAGES += \
     copybit.msm8974 \
-- 
2.13.2.windows.1

